docs/docs/cloud/aws/compliance.md

```diff
@@ -0,0 +1,258 @@
+# AWS Compliance
+
+## CIS Compliance Report
+
+!!! warning "EXPERIMENTAL"
+This feature might change without preserving backwards compatibility.
```

Comment:
I think you need spaces or tabs here.

<img width="697" alt="image" src="https://user-images.githubusercontent.com/2253692/205479504-3ca06263-f60d-431f-9c10-a9205a2f6885.png">

Reply:
Fixed in https://github.com/aquasecurity/trivy/pull/3130/commits/813d6e9e997acf4c656462688321bcdd23b84869

Reply:
thanks for catching that!